app/Category.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    protected $table = 'categories';
+
+    protected $fillable = [
+    	'name', 'parent_id',
+    ];
+
+    public function products()
+    {
+    	return $this->hasMany('App\Product');
```

Comment:
return $this->hasMany(Product::class);
fix tương tự cho những chỗ khác. Mấy cái này anh có comment ở quick task rồi mà nhỉ?

Reply:
trước chưa có nơi anh